### DISCUSSION TITLE
Future for C# Support

### DISCUSSION THREAD

**Discussion by OmarVector** [sqlitecloud/discussions#35] [SDK] [answered]
Is this planned to be added?

I tried to use System.Data.SqlClient , yet the connection string never worked... and I'm not sure if its bec you are not supported yet C# or I did something wrong

Generally would love to hear more regard the future for C# support and Unity

Thanks:)

**Comment by Gioee** [comment, marked as answer]
Hi @OmarVector 
Thank you for the question.

We're prioritizing mobile devs support first and C# is not currently in our priorities.

That namespace you asked about is strictly for SQL Server by Microsoft https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient

I'm flagging this issue as duplicate of https://github.com/orgs/sqlitecloud/discussions/43
